scrapers_next/nv/people.py

```diff
@@ -3,8 +3,13 @@
 import re
 
 
+class NewDetailFieldEncountered(BaseException):
+    pass
+
+
 class Legislators(HtmlListPage):
     selector = CSS("tbody tr.thisRow.listRow")
+    field_names = set()
```

Comment:
This was accidentally left in. I will push up a commit to remove this set creation.

Reply:
Just gotta do that & looks good